src/Http/Controllers/V1/Shop/Customer/WishlistController.php

```diff
@@ -78,7 +78,7 @@ public function addOrRemove(Request $request, $id)
             $this->wishlistRepository->delete($wishlistItem->id);
 
             return response([
-                'data'    => null,
+                'data'    => CustomerWishlistResource::collection($customer->wishlist_items()->get()),
```

Comment:
I think we should not mix this. If someone wants the remaining data, they can simply hit another API as this causes another query to the database.

Reply:
@devansh-webkul 

This is the valid PR, no need to hit another API if we are getting the data in the same API.